contracts/MerkleTreeUtils.sol

```diff
@@ -115,87 +138,6 @@ contract MerkleTreeUtils {
         return nodes[0];
     }
 
-    /**
-     * @notice Calculate root from an inclusion proof.
-     * @param _dataBlock The data block we're calculating root for.
-     * @param _path The path from the leaf to the root.
-     * @param _siblings The sibling nodes along the way.
-     * @return The next level of the tree
-     * NOTE: This is a stateless operation
-     */
-    function computeInclusionProofRoot(
-        bytes memory _dataBlock,
-        uint256 _path,
-        bytes32[] memory _siblings
-    ) public pure returns (bytes32) {
-        // First compute the leaf node
-        bytes32 computedNode = keccak256(_dataBlock);
-
-        for (uint256 i = 0; i < _siblings.length; i++) {
-            bytes32 sibling = _siblings[i];
-            uint8 isComputedRightSibling = getNthBitFromRight(_path, i);
-            if (isComputedRightSibling == 0) {
-                computedNode = getParent(computedNode, sibling);
-            } else {
-                computedNode = getParent(sibling, computedNode);
-            }
-        }
-        // Check if the computed node (_root) is equal to the provided root
-        return computedNode;
-    }
-
-    /**
-     * @notice Calculate root from an inclusion proof.
-     * @param _leaf The data block we're calculating root for.
-     * @param _path The path from the leaf to the root.
-     * @param _siblings The sibling nodes along the way.
-     * @return The next level of the tree
-     * NOTE: This is a stateless operation
-     */
-    function computeInclusionProofRootWithLeaf(
```

Comment:
computeInclusionProofRootWithLeaf is basically computeInclusionProofRoot 😜 and is replaced by rootFromWitnesses too.